templates/engage/fr/vmware-to-openstack.html

```diff
@@ -0,0 +1,21 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}Migrer de VMware à Openstack{% endblock %}
+
+{% block meta_description %}Quels sont les différences entre VMWare et OpenStack et les bénéfices d'une migration de l'un vers l'autre. Un webinar gratuit pour mieux comprendre les technologies du cloud privé.{% endblock %}
```

Comment:
@caldav 	should this be a question?

`Quels sont les différences entre VMWare et OpenStack et les bénéfices d'une migration de l'un vers l'autre ?`

Reply:
The idea is to rank #1 for people googling the question, but since we are actually answering it, it's more assertive to skip the question mark.